### DISCUSSION TITLE
Is it possible to hide the cursor when on alpha?

### DISCUSSION THREAD

**Discussion by LudoPinelli** [goolord/alpha-nvim#75] [Q&A]
I'm sure there is a way, but I'm pretty new to the lua/neovim/programming world and can't find an easy answer. 
It's a detail but I find that having this block hiding the first letter on the landing page is pretty ugly ^^
![alpha](https://user-images.githubusercontent.com/93927273/148225678-8ee446b4-42cd-43e9-a482-04e3849951d0.png)

**Comment by geodimm** [comment]
Try that :) Adapted from https://github.com/neovim/neovim/issues/3688#issuecomment-574544618

```lua
local config = {
  layout = {
    ...
  },
  opts = {
    setup = function()
      vim.api.nvim_create_autocmd('User', {
        pattern = 'AlphaReady',
        desc = 'hide cursor for alpha',
        callback = function()
          local hl = vim.api.nvim_get_hl_by_name('Cursor', true)
          hl.blend = 100
          vim.api.nvim_set_hl(0, 'Cursor', hl)
          vim.opt.guicursor:append('a:Cursor/lCursor')
        end,
      })
      vim.api.nvim_create_autocmd('BufUnload', {
        buffer = 0,
        desc = 'show cursor after alpha',
        callback = function()
          local hl = vim.api.nvim_get_hl_by_name('Cursor', true)
          hl.blend = 0
          vim.api.nvim_set_hl(0, 'Cursor', hl)
          vim.opt.guicursor:remove('a:Cursor/lCursor')
        end,
      })
    end,
    ...
  },
}
```

**Comment by anuramat** [comment]
just in case anyone else stumbles upon this, there's another way
if your terminal emulator uses the inverse pair of cell colors for the cursor, you can use a fake button with a U+2588 block to effectively hide the cursor:
```lua
    { type = 'button', val = '█' },
```
the only downside is that the cursor becomes visible again when the terminal is out of focus

**Reply by alaanvv** [reply]
what do you mean by using a fake cursor? where should I place that

**Reply by anuramat** [reply]
it's a button that does nothing
just put it in the beginning of your menu

**Reply by goolord** [reply]
lol, awesome solution
